### DISCUSSION TITLE
Please help me to singers

### DISCUSSION THREAD

**Discussion by maxcell1024** [NAIST-SE/AttractingContributors#28] [General]
Please help me with my singer list.
https://github.com/Maxcell1024/AttractingContributors-Singer

**Comment by vittunyuta-aom** [comment]
added!!

**Comment by haxhimitsu** [comment]
Thanks your merge:)

**Comment by nosfill** [comment]
I added

**Comment by eedy8888** [comment]
added please help me
https://github.com/eedy8888/AttractingContributors
